font/ttf.ksy

```diff
@@ -242,6 +242,112 @@ types:
                 if: repeat
+  gvar:
+    doc: >
+      gvar - Glyph Variations Table
+      This table provides all of the variation data that describe how TrueType glyph outlines in a 'glyf' table change across the font’s variation space.
+    seq:
+      - id: major_version
+        type: u2
+      - id: minor_version
+        type: u2
+      - id: axis_count
+        type: u2
+      - id: shared_tuple_count
+        type: u2
+      - id: shared_tuples_offset
+        type: u4
+      - id: glyph_count
+        type: u2
+      - id: flags
+        type: u2
+      - id: glyph_variation_data_array_offset
+        type: u4
+      - id: glyph_variation_data_offsets
+        type:
+          switch-on: flags & 0x01
+          cases:
+            0: u2
+            1: u4
+        repeat: expr
+        repeat-expr: glyph_count + 1
+    instances:
+      shared_tuples:
+        pos: shared_tuples_offset
+        type: tuple_record
+        repeat: expr
+        repeat-expr: shared_tuple_count
+      glyph_variation_data:
+        #The body of glyph_variation_datum should go here
+        #See https://github.com/kaitai-io/kaitai_struct/issues/436
+        type: glyph_variation_datum(_index)
+        repeat: expr
+        repeat-expr: glyph_count
+    types:
+      f2dot14:
+        seq:
+          - id: raw_coordinate
+            type: s2
+        instances:
+          coordinate:
+            value: raw_coordinate / 16384.0
+            -webide-parse-mode: eager
+        -webide-representation: "{coordinate:dec}"
+      tuple_record:
+        seq:
+          - id: coordinates
+            type: f2dot14
+            repeat: expr
+            repeat-expr: _parent.axis_count
+      tuple_variation_header:
+        seq:
+          - id: variation_data_size
+            type: u2
+          - id: tuple_index
+            type: u2
+          - id: peak_tuple
+            type: tuple_record
+            parent: _parent
+            if: (tuple_index & 0x8000) != 0
+          - id: intermediate_start_tuple
+            type: tuple_record
+            parent: _parent
+            if: (tuple_index & 0x4000) != 0
+          - id: intermediate_end_tuple
+            type: tuple_record
+            parent: _parent
+            if: (tuple_index & 0x4000) != 0
+      tuple_variation_store_header:
+        seq:
+          - id: tuple_variation_count
+            type: u2
+            if: _io.size > 0
+          - id: data_offset
+            type: u2
+            if: _io.size > 0
+          - id: tuple_variation_headers
+            type: tuple_variation_header
+            parent: _parent
+            repeat: expr
+            repeat-expr: tuple_variation_count & 0x0FFF
+            if: _io.size > 0
+        instances:
+          #The serialized data interpretation depends on each of the tuple_variation_header.tuple_index flags
+          #https://docs.microsoft.com/en-us/typography/opentype/spec/otvarcommonformats#a-nameserializeddata-idserializeddataserialized-dataa
```

Comment:
use `doc` and `-doc-ref`

Reply:
This was more an implementation comment as to why this isn't complete, I suppose it could end up in a doc for now.

Reply:
Done